examples/simple-genai-server/gameserver_manualchat.yaml

```diff
@@ -44,10 +44,10 @@ spec:
             limits:
               memory: 64Mi
               cpu: 20m
-      # Schedule onto the game server node pool when running in the same cluster as the inference server.
-      # tolerations:
-      #   - key: "agones.dev/role"
-      #     value: "gameserver"
-      #     effect: "NoExecute"
-      # nodeSelector:
-      #   agones.dev/role: gameserver
+              # Schedule onto the game server node pool when running in the same cluster as the inference server.
```

Comment:
I think this ended up in by mistake?

Reply:
Sorry if I wasn't clear - this should be reverted as well.

Reply:
Modified this file to fix the lint warning. Reverted this change